test/patronagebadges.js

```diff
@@ -73,28 +81,52 @@ contract('Auto Badges', (accounts) => {
     assert.isTrue(checkHexEquality(web3.utils.toHex(computedID3), jscomputedid3));
     assert.isTrue(checkHexEquality(web3.utils.toHex(computedID4), jscomputedid4));
 
+    assert.equal(await badges.totalSupply(), 4);
+    await badges.burnToken(computedID1);
+    assert.equal(await badges.totalSupply(), 3);
+  });
```

Comment:
This still needs more based on the test above.

Mint 5
Burn 3
Mint new 3

Ran into an issue previously where I didn't catch this properly. So it necessary to do this in sequence. :)